pkg/dash/config.go

```diff
@@ -15,29 +15,22 @@ type Config struct {
 // ParseEnvironment parses necessary environment variables
 func ParseEnvironment() (*Config, error) {
 	var config Config
-	if url, exists := os.LookupEnv("GRAFANA_URL"); exists {
-		config.GrafanaURL = url
-	} else {
-		protocol, protocolExists := os.LookupEnv("GRAFANA_PROTOCOL")
-		user, userExists := os.LookupEnv("GRAFANA_USER")
-		token, tokenExists := os.LookupEnv("GRAFANA_TOKEN")
-		host, hostExists := os.LookupEnv("GRAFANA_HOST")
-		path, pathExists := os.LookupEnv("GRAFANA_PATH")
-		if !hostExists {
-			return nil, errors.New("Either GRAFANA_URL or GRAFANA_HOST required")
-		}
-		if !protocolExists {
-			protocol = "https"
-		}
-		auth := ""
-		if userExists && tokenExists {
-			auth = fmt.Sprintf("%s:%s", user, token)
+	if gu, exists := os.LookupEnv("GRAFANA_URL"); exists {
```

Comment:
```suggestion
	if grafanaURL, exists := os.LookupEnv("GRAFANA_URL"); exists {
```
might be a bit more descriptive :-)

Reply:
Yeah, "gu" is a little cryptic. Go folks seem to always want to encourage short variable names. For example, https://github.com/golang/go/wiki/CodeReviewComments#variable-names. Being that the declaration and usage are a line apart, I figured I'd make it short as possible.